cmd/sovereignnode/chainSimulator/tests/bridge/bridge_test.go

```diff
@@ -108,10 +105,113 @@ func TestSovereignChainSimulator_DeployBridgeContractsThenIssueAndDeposit(t *tes
 	numDecimals := 18
 	tokenIdentifier := chainSim.IssueFungible(t, cs, wallet.Bytes, &nonce, issueCost, tokenName, tokenTicker, numDecimals, supply)
 
+	depositAndCheckTokens(t, cs, wallet, nonce, bridgeData, tokenIdentifier, supply)
+
+	// Wait for outgoing operations to get unconfirmed and check we have one, which is also saved in storage
+	time.Sleep(time.Second)
+
+	checkOutGoingOperation(t, cs)
+}
+
+// This test will:
+// - deploy bridge contracts setup
+// - generate new wallet and set a token without prefix (a.k.a main chain token)
+// - deposit the token in esdt-safe contract
+// - check the sender balance is correct
+// - check the token burned amount is correct after deposit
+func TestSovereignChainSimulator_DeployBridgeContractsAndDepositMainChainToken(t *testing.T) {
```

Comment:
[optional]
So this test "manually" -aka `SetKeyValueForAddress` + `SetEsdtInWallet` creates this main chain token in the sovereign chain as if it was previously bridged from main->sov.

Wouldn't it be a more interesting test to bring those main chain tokens first from main->sov, instead of "artifically" planting them here?

Just a thought, could've made the test more "real-life", could be done in another PR if you see it suitable